docs/source/community/persons_of_interest.rst

```diff
@@ -334,7 +370,7 @@ TorchAudio
 ~~~~~~~~~~
 
 -  Moto Hira (`mthrok <https://github.com/mthrok>`__)
```

Comment:
@mthrok are you still maintaining TorchAudio?

Reply:
Keeping as is for now, we can update in a follow up if needed